### DISCUSSION TITLE
My personal github account was linked to another account created with mail.

### DISCUSSION THREAD

**Discussion by DavidVidaurre** [vercel/community#3742] [Account Issue] [answered]
### Question

I was provided with a vercel account with mail and I did the login, when I was going to upload a project from a repository this was linked to my personal github account that already had my account created with vercel, so now when I login to vercel with my personal github I get the account with mail and my projects that I had before do not appear.

**Comment by amyegan** [comment, marked as answer]
Hey @DavidVidaurre. Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. Please email [registration@vercel.com](mailto:registration@vercel.com) for help with login issues.
